seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/TableSchema.java

```diff
@@ -68,6 +72,10 @@ public static final class Builder {
 
         private final List<ConstraintKey> constraintKeys = new ArrayList<>();
 
+        private final List<VectorIndex> vectorIndexes = new ArrayList<>();
+
+        private Boolean enableDynamicField;
```

Comment:
Please add some comment to tell other developer what's used for? And how/when to use it?